src/bikeracks/templates/scripts.js

```diff
@@ -627,14 +605,14 @@ BikeMap.prototype.popupContent = function(state) {
                <div id="options">`
     
     // if user is online and this isn't a temporary marker include the submit button and the voting buttons
-    if (onlineStatus && state.status) {
+    if (onlineStatus && state.rack_id) {
```

Comment:
`state.rack_id` is another example of a magical check with implicit meaning behind it.

This could be `if (isLoggedIn && state.isTemporary())`, or something similar, which is self explanatory.

Reply:
Done.